lib/keystore/helpers.go

```diff
@@ -112,11 +112,19 @@ func GenerateKeypair(keytype string, kp crypto.Keypair, basepath string, passwor
 }
 
 // LoadKeystore loads a new keystore and inserts the test key into the keystore
-func LoadKeystore(key string, ks Keystore) error {
+func LoadKeystore(key string, ks TyperInserter) (err error) {
```

Comment:
change to a fast return if `key == ""`
```go
func LoadKeystore(key string, ks TyperInserter) (err error) {
  if key == "" { return nil }
  ...
}

Reply:
Done in #2886 to move this if check up the call stack